doc/howto/generic-install-from-source.rst

```diff
@@ -4,7 +4,7 @@
 
 This is a generic guide to installing NAV from source code on a \*NIX flavored
 operating system. The specifics of how to install NAV's dependencies, such as
-PostgreSQL_ or Graphite_ will be entirely up to you and your choice of OS.
+:xref:`PostgreSQL` or :xref:`Graphite` will be entirely up to you and your choice of OS.
```

Comment:
This is one of the reasons Restructured text rocks, adding in new tokens like this. Real syntax FTW!